File: watch-app/SiWatchApp/Models/MonitorRecord.cs
using SiWatchApp.Monitors;

namespace SiWatchApp.Models
{
    public class MonitorRecord : Record
    {
        public MonitorType MonitorType { get; set; }

        public MonitorRecord() { }

        public MonitorRecord(MonitorType monitorType, object monitorValue)
        {
            MonitorType = monitorType;
            Value = monitorValue;
        }
        
        public override string ToString()
        {
            return $"MonitorRecord{{MonitorType={MonitorType},Timestamp={Timestamp},Value={Value}}}";
        }
    }
}

File: backend/facade/siwatch-device-api/src/main/java/ru/nord/siwatch/backend/facade/device/models/monitoring/MonitorRecord.java
package ru.nord.siwatch.backend.facade.device.models.monitoring;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDateTime;

@Getter @Setter @ToString
public class MonitorRecord
{
    private MonitorType monitorType;
    private LocalDateTime timestamp;
    private Object value;
}
